prow/config/config.yaml

```diff
@@ -129,6 +129,63 @@ branch-protection:
                   - "unit-test"
                   - "license/cla"
                 strict: true
+        tidb:
+          branches:
+            master:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                strict: true
+            release-2.1:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                strict: true
+            release-3.0:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                strict: true
+            release-4.0:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                strict: true
+            release-5.0-rc:
```

Comment:
we'll need release 5.0 soon

Reply:
We'll configure it later. This configuration can be updated at any time, and the PR merge will automatically update to the bot k8s cluster to take effect.